contrib/cirrus/setup_environment.sh

```diff
@@ -208,7 +222,7 @@ case "$TEST_FLAVOR" in
     unit) ;;
     apiv2) ;&  # use next item
     compose)
-        dnf install -y $PACKAGE_DOWNLOAD_DIR/podman-docker*
+        rpm -ivh $PACKAGE_DOWNLOAD_DIR/podman-docker*
```

Comment:
This is the only eyebrow-raiser I found: it could fail if a dependency of `podman-docker` is missing. I choose not to worry about it.

Reply:
This is accounted for when the packages are downloaded.  This same 'install from cache' mechanism is used in CI for other containers repos, so we'll notice pretty quickly if there's a problem.  I think it's fine to not worry about it.